packages/analytics/amplify_analytics_pinpoint_dart/lib/src/version.dart

```diff
@@ -1,2 +1,2 @@
 // Generated code. Do not modify.
-const packageVersion = '0.2.0+1';
+const packageVersion = '0.2.0+2';
```

Comment:
Sanity check - Is this used for generating user-agent? If yes, should this be located under `amplify_analytics_pinpoint` with version `1.0.0`?

Reply:
It's not used in the user agent but in the event's sdkVersion field. @fjnoyp please confirm or adjust.

We do want to support both Dart and Flutter usages so it's okay for this package to embed it's version as long as the Flutter package overrides it if needed.